spec/requests/api/admin/create_spec.rb

```diff
@@ -18,6 +28,11 @@
     it 'returns a 200 response status' do
       expect(response).to have_http_status 200
     end
+
+    it 'returns an attached image' do
```

Comment:
why would it return an image?

Reply:
ah yeah. Thats poor wording there, has been changed.